homeassistant/components/homekit_controller/switch.py

```diff
@@ -18,6 +24,30 @@
 ATTR_REMAINING_DURATION = "remaining_duration"
 
 
+@dataclass
+class DeclarativeSwitchEntityDescription(SwitchEntityDescription):
+    """Describes Homekit button."""
+
+    true_value: bool = True
```

Comment:
I'm assuming this is for the inconsistency with HAP where some accessories want 1 and 0?

Reply:
I think aiohomekit actually translates bools to 1 and 0 because of misbehaving HAP devices. But yes, this is just future proofing / consistency with button.py because i expect some weird ones going forwards. We have something similiar for one device in Button where the value to write is some weird string of characters like `@trxcmd` or something.